aten/src/ATen/cuda/nvrtc_stub/ATenNVRTC.h

```diff
@@ -84,7 +84,9 @@ namespace at { namespace cuda {
   _(nvrtcCompileProgram)                         \
   _(cuCtxGetCurrent)                             \
   _(cuModuleUnload)                              \
-  _(cuDevicePrimaryCtxGetState)
+  _(cuLinkCreate)                                \
```

Comment:
`cuDevicePrimaryCtxGetState` is deleted by mistake.

Reply:
Reverted ATenNVRTC.h